submissions/chabuChabu/css/contact-styles.css

```diff
@@ -0,0 +1,334 @@
+*{
+	box-sizing: border-box;
+}
+html, body{
+	height:100%;
+	padding: 0;
+	margin: 0;
+	background: #f6f6f6;
+	font-family: 'Poppins', sans-serif;
+}
+a{
+   text-decoration: none;
+}
+header{
+	background-color: #69629f;
+	height: 66px;
+}
+header *{
+	color: white;
+}
+header .logo{
+	float: left;
+	height: inherit;
+	margin-left: 20px;
+}
+
+header .logo-text{
+	margin: 9px;
+}
+header ul {
+	float: right;
+	margin: 0
```

Comment:
There is a missing semicolon here @ChabuChabu

Reply:
Done